notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb

```diff
@@ -0,0 +1,754 @@
+{
```

Comment:
<p>Small suggestion on the second set of bullet points:</p><ul><li>Second point, suggestion rewording: "...qmin, the mass ratio of the companion to the primary star,"</li></ul><p> </p><p>At the end of this section, you might also want to just put a note along the lines of "For details on the derivation of these parameters, please refer to the Price-Whelan et al. 2018 paper."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I think it's important to keep "minimum," but I did make the change from "or" to "the." </p>